.github/workflows/test.yml

```diff
@@ -16,11 +19,11 @@ jobs:
 
     services:
       postgres:
-        image: postgres:12
+        image: postgres:13-alpine
```

Comment:
Hey @foarsitter , sorry for bringing this up again. This is by no means nitpicking, just a honest question.

Is there any reason to use version 13 instead of the latest available ([16-alpine](https://hub.docker.com/layers/library/postgres/16-alpine/images/sha256-6c070fa97f2e655563c228b7ff1db4b55a1abc338c64cf739a9099812e2576a9?context=explore)) ?

Reply:
13 is the lowest version compatible with main: https://docs.djangoproject.com/en/dev/ref/databases/#postgresql-notes

The docs don't mention an upper limit so Im not sure if 4.2 is compatible with 16. Although our needs are very basic so probably will.

Reply:
I see. I agree with your decision :)